contracts/signature/ERC1271Stored.sol

```diff
@@ -11,7 +11,7 @@ import './ERC1271StoredStorage.sol';
 abstract contract ERC1271Stored is ERC1271Base {
   function _isValidSignature (
     bytes32 hash,
-    bytes memory
+    bytes memory signature
```

Comment:
Why name the variable?  It's unused and therefore will cause a compilation warning.

Reply:
Not having this variable named prevents typechain from generating the typed contracts.

![image](https://user-images.githubusercontent.com/7013086/119603892-b03dc080-be20-11eb-962b-cec78dea697f.png)

Reply:
Do you know of a fundamental reason for this, or should I open an issue in the TypeChain repository?

Reply:
For now I think the solution is to add a line `signature = signature`.  I don't think this costs gas if the optimizer is turned on, though not sure.

Reply:
No, not aware of a particular reason for this